### DISCUSSION TITLE
Developing the WordPress.org YouTube channel

### DISCUSSION THREAD

**Discussion by sereedmedia** [WordPress/Marketing-Team#309] [General]
This GitHub discussion serves as a place to brainstorm ideas on how the Global Make WordPress Marketing Team can develop the official WordPress.org channels.

Please also feel free to post any ideas generally related to the broader umbrella topic of developing WordPress video content, strategy, and quality. We will branch topics into new discussions or issues as necessary.

**Comment by meherbala** [comment]
@sereedmedia not sure if you know but in the past WordPress Marketing had created its own YouTube Channel. 

@abhansnuk can give more backstory about this.

Here is a link to the channel: https://www.youtube.com/@WordPressMarketingTeam

In the channel, there are some videos that are linked either in Learn or in the Marketing Handbook.

**Comment by sereedmedia** [comment]
@meherbala I did not know that, thank you!

**Reply by meherbala** [reply]
Most welcome. It was supposed to be added to Marketing Handbook but I guess it got missed.

**Comment by abhansnuk** [comment]
It was set up with specific permissions, including for interim assistance to WCEU contributor day / contributor marketing efforts/ support for polyglots and onboarding, and for live sessions for polyglots / WPTranslationDay.

It was not to replace the main WPTV channel, and items were identified that could be used on Learn WP (which came later) and for once edited going onto WPTV.

________________________________

**Comment by eidolonnight** [comment]
We now use the [WPTV to YT Publication Process](https://make.wordpress.org/marketing/handbook/social-media/wptv-to-yt-publication-process/) to automatically bring WPTV videos to YouTube.
